src/runner.js

```diff
@@ -1,4 +1,24 @@
- /* istanbul ignore next */
-require('babel-core/register');
+/* istanbul ignore next */
+var fs = require('fs');
+/* istanbul ignore next */
+var path = require('path');
+
+/* istanbul ignore next */
+var babelrc = fs.readFileSync(path.resolve(__dirname, '../.babelrc'));
```

Comment:
Ooo, why we need to load babelrc and parse it to config and pass it to the babel-core/register?